src/main/java/ru/sbtqa/tag/pagefactory/support/DataTableParser.java

```diff
@@ -0,0 +1,16 @@
+package ru.sbtqa.tag.pagefactory.support;
+
+import cucumber.api.DataTable;
+import java.util.List;
+
+public class DataTableParser {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)